include/nle.h

```diff
@@ -7,6 +7,8 @@
 
 #include <fcontext/fcontext.h>
 
+#include "tmt.h"
```

Comment:
We can _probably_ get by with a `typedef struct TMT TMT;` here and include `tmt.h` in nle.c only. I'm not a C guru but that strikes me as slightly preferable?